topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,481 @@
+The study investigated how the intestinal microbiome of children with cystic fibrosis (CF) evolves over time while being treated with the elexacaftor-tezacaftor-ivacaftor (ETI) combination therapy. MaAsLin2 was employed to detect shifts in microbial abundance associated with this treatment. By performing longitudinal microbiome profiling, the research tracked changes in gut microbial diversity and composition in response to ETI treatment. The findings highlighted significant shifts in the microbiome, which could impact gut health and inform future CF treatment approaches. 
+MaAsLin2 was set up with the following parameters:
+
+1. **fixed effects**: treatment(ELX/TEZ/IVA), age, and recent antibiotic exposure
+2. **random effects**: Subject ID was specified as a random effect due to multiple samples from the same subject
+3. **min_prevalence**: The minimum prevalence threshold was set to 0.1, indicating that features must be present in at least 10% of the samples to be included.
+4. **transform**: LOG transformed
+5. **Analysis method**: The general linear “LM” model was used.
+6. **Correction method**: The Benjamini-Hochberg procedure was used to correct P values
+7. **Normalization method**: A Centered Log-Ratio (CLR) normalization approach was used instead of default normalization methods
+
+- **The infant gut resistome is associated with E. coli and early-life exposures** ({% cite Lebeaux2021 %}):\
+The study investigated how the infant gut resistome—the collection of antibiotic resistance genes (ARGs) in the gut microbiome—associates with E. coli and early-life exposures using MaAsLin2.
+The analysis, which utilized additive boosting of generalized linear models for feature reduction, revealed significant associations between ARGs and E. coli presence, as well as early-life factors such
+as antibiotic use and other exposures. Key parameters included CLR normalization of compositional abundance data, no standardization of continuous variables, and a strict significance threshold
+(q-value < 0.01) using Benjamini-Hochberg correction. This approach highlighted how early exposures influence the resistome and its relationship with E. coli in the infant's gut.
+
+# Conclusion
+
+In essence, uncovering associations between microbial features and metadata variables through tools like MaAsLin2 not only deepens our understanding of microbiome dynamics but also holds promise for clinical applications, personalized health strategies, and advancing the field of microbiome research.
+
+The results obtained from MaAslin2 can further be visualized using tools like [**phyloseq**](https://training.galaxyproject.org/training-material/by-tool/interactive_tool_phyloseq.html).\
+Tools like phyloseq require that you prepare your data (OTU/ASV table, metadata) in a structured format before visualizing.\
```

Comment:
I would add here, that the features need to be subsampled to only include significant features. This can be done with https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fampvis2_subset_taxa%2Fampvis2_subset_taxa%2F2.8.9%2Bgalaxy0&version=latest